scripts/test_flow.sh

```diff
@@ -0,0 +1,216 @@
+  stop_node
+  echo "Cleanup"
+
+  if [[ -z "$1" ]]; then
+    exit 0
+  else
+    exit "$1"
+  fi
+}
+
+error_exit() {
+  finish 1
+}
+
+assert_eq() {
+  a=$(echo "$1" | tr -d '"')
+  b=$(echo "$2" | tr -d '"')
+
+  if [[ "$a" != "$b" ]]; then
+    echo "Unexpected result, should be $1 but actual is $2"
+    finish 1
+  fi
+}
+
+check_env_var() {
+  if [ -z "$(eval echo \$$1)" ]; then
+      echo "$1 environment variable doesn't exist."
+      error_exit
+  fi
+}
+
+download_aurora_contract() {
+  curl -sL $ENGINE_LAST_WASM_URL -o $ENGINE_WASM || error_exit
+}
+
+get_aurora_and_build() {
+  curr_dir=$(pwd)
+  rm -rf /tmp/aurora-contract
+  git clone https://github.com/aurora-is-near/aurora-engine.git /tmp/aurora-contract > /dev/null 2>&1
+  cd /tmp/aurora-contract || error_exit
+  # cargo make --profile=mainnet build-test > /dev/null 2>&1 || error_exit
+  RUSTFLAGS="-C link-arg=-s" cargo build --target wasm32-unknown-unknown --release --no-default-features --features=mainnet,integration-test -p aurora-engine -Z avoid-dev-deps > /dev/null 2>&1 || error_exit
+  cd $curr_dir || error_exit
+}
+
+build_migration_tool() {
+  cd ..
+  cargo build --features localnet --release > /dev/null 2>&1 || error_exit
+  cd scripts || error_exit
+}
+
+get_eth_connector_and_build_for_migration() {
+  curr_dir=$(pwd)
+  rm -rf /tmp/aurora-eth-connector
+  git clone https://github.com/aurora-is-near/aurora-eth-connector.git /tmp/aurora-eth-connector  > /dev/null 2>&1
+  cd /tmp/aurora-eth-connector || error_exit
+  cargo make --profile=mainnet build-test > /dev/null 2>&1 || error_exit
+  cd $curr_dir || error_exit
+}
+
+get_aurora_contract_state() {
+  http post http://127.0.0.1:3030 jsonrpc=2.0 id=dontcare method=query \
+    params:='{
+      "request_type": "view_state",
+      "finality": "final",
+      "account_id": "'"$ENGINE_ACCOUNT"'",
+      "prefix_base64": ""
+    }' > res_state.json
+}
+
+#==================================
+# Main
+
+rm -rf /tmp/localnet
+
+echo "Install nearup"
+install_nearup
+
+echo "Start NEAR node"
+start_node
+sleep 2
+
+echo "Get and build Aurora contract"
+get_aurora_and_build
+
+export NEAR_KEY_PATH=$NODE_KEY_PATH
+echo "Create account for Aurora"
+aurora-cli create-account --account $ENGINE_ACCOUNT --balance 1000 > $AURORA_KEY_PATH || error_exit
+sleep 2
+
+echo "View info of created Aurora account"
+balance=$(aurora-cli view-account $ENGINE_ACCOUNT  | jq '.amount') || error_exit
+sleep 1
+assert_eq "$balance" "1000000000000000000000000000"
+echo $balance
+
+export NEAR_KEY_PATH=$AURORA_KEY_PATH
+echo "Deploy Aurora contract"
+aurora-cli deploy-aurora $ENGINE_WASM || error_exit
+sleep 4
+
+echo "Init Aurora"
+aurora-cli --engine $ENGINE_ACCOUNT init \
+  --chain-id 1313161556 \
+  --owner-id $ENGINE_ACCOUNT \
+  --bridge-prover-id "aurora.node0" \
+  --upgrade-delay-blocks 1 \
+  --custodian-address 0x096DE9C2B8A5B8c22cEe3289B101f6960d68E51E \
+  --ft-metadata-path engine_ft_metadata.json || error_exit
```

Comment:
I believe you can leave the `--chain-id`, `--owner-id` and `--upgrade-delay-blocks` args. Other are already outdated?